### DISCUSSION TITLE
Q: motd input format

### DISCUSSION THREAD

**Discussion by mwpowellhtx** [Foxlider/FASTER#146] [Q&A]
How to input that?

![image](https://github.com/Foxlider/FASTER/assets/1824187/885edb13-3256-43ce-8ba1-09ff38afe643)

i.e. (?)

```
/*
    Two lines welcome message. Comma is the 'new line' separator.
 */
motd[] = {
    "line one..."
    , "line two..."
};
```

**Comment by jupster** [comment]
Did you check if what you have current works? 

Edit: plus did you check the documentation for it?

**Reply by mwpowellhtx** [reply]
This is the [docs](https://github.com/Foxlider/FASTER/wiki/Profile-Page#motd) for it.

> Message of the Day Sets the message of the day displayed at server start or when users log in
> Delay Delay in seconds between each MOTD lines

Delay _self explanatory_. No mention of the _multiple lines_ part of the question.

**Comment by MildlyInterested** [comment]
Each line is a message sent with the configured delay below. You can check that by saving in the GUI and looking at the resulting server.cfg.
No special characters needed, FASTER takes care of that.
```
Line 1
Line 2
etc
```

**Reply by OverlordZorn** [reply]
As an Example:
![grafik](https://github.com/Foxlider/FASTER/assets/56258612/89456111-238d-4373-89e2-ae5f423fdb15)

**Reply by mwpowellhtx** [reply]
Ah okay and which expanding the output, I see:

```
// WELCOME MESSAGE
motd[] = {
	"line one...",
	 "line two..."
};
```

Very good, very good. 👍
